build.json

```diff
@@ -2,7 +2,7 @@
   "DotNetInstallScriptURL": "https://dot.net/v1",
   "DotNetChannel": "preview",
   "DotNetVersion": "2.1.505",
-  "RequiredMonoVersion": "5.18.0.0",
+  "RequiredMonoVersion": "6.0.0",
```

Comment:
a short explanation why this isn't 6.4.0 here - there is no 6.4.0 for travis yet (unless we customize the build and install manually which I want to avoid) so the build would fail.

It doesn't really matter since on ADO we explicitly set 6.4.0 and build with that version, and this is where the real "publishing" happens. Plus we have #1569 pending and at that point we can stop worrying about travis too.

As soon as #1569 is done or travis has 6.4.0, we can raise the version in this file too.

Reply:
So what you're saying is you want me to disconnect travis and appveyor.

Reply:
exactly, I think it adds maintenance overhead unnecessarily - and we really can get (and we do get) everything done via Azure DevOps already anyway

Reply:
> Assuming this passes CI :shipit:
> 
> This may increase the size of the output assets, so we should review with that @rchande and @akshita31 to make sure it's too much larger when consumed from the vscode plugin.

I checked the outputs from the current master and this change... and the was no real large change in overall zip size.  It looked to be about 50KB at most, so